### ISSUE TITLE
Convert nullable string properties to non-nullable at `AccessibleObject`

### ISSUE THREAD

**Issue by ArtemTatarinov** [dotnet/winforms#6103]
Some properties at the `AccessibleObject` class have a nullable `string?` type, like: `DefaultAction`, `Description`, `Help`, `KeyboardShortcut`, `Name`, `Value`. We're trying to avoid `null` values and return `string.Empty` instead. We should convert these properties to non-nullable.

_Originally posted by @dreddy-work in https://github.com/dotnet/winforms/pull/5745#discussion_r740372801_

**Comment by NikitaSemenovAkvelon** [comment]
I suggest closing this issue. The reasons are:

1. We cannot change the type of any properties without changing in daughter classes. 
2. Why `null` is an incorrect value? And why `string.Empty` is better?
3. Now, we have a single null-check inside `GetPropertyValue` method. After refactoring, we will have it in many places. Codebase will be grown after such refactoring. Without any profit as I can see.

@dreddy-work what do you think about that points?

**Comment by Tanya-Solyanik** [comment]
`null` and `string.Empty` have different meanings, one means defer to the native proxy and the other means "show an empty string in inspect". It does not make sense to unify strings

**Comment by dreddy-work** [comment]
I have not looked at other properties. Comment was specific to `Help`.  Base class property should have been default to empty string (since that is what we are returning) and non-nullable. Derived classes can override whichever the way make it more opt.

I am fine punting this given this is more of a syntactic change at this point.

**Comment by NikitaSemenovAkvelon** [comment]
Closed as not planned.
